src/components/GameComponent.tsx

```diff
@@ -28,18 +28,37 @@ const config = {
   parent: 'main-game',
   scene: [FortNerf],
 };
-type killMe = {
+type styleProps = {
   width: string;
 };
-function GameComponent({ width }: killMe) {
+function GameComponent({ width }: styleProps) {
+  const canvasRef = useRef<any>();
```

Comment:
Be sure to to go back eventually and correctly type anyplace you're using type `any`.